src/components/Tooltip/TooltipCard.js

```diff
@@ -0,0 +1,67 @@
+import { formatAmount, USD_DECIMALS } from "../../lib/legacy";
+
```

Comment:
1. Hm, I think the name Tooltip card is too abstract here, i mean it sounds like a general tooltip for any content like onboarding, errors, etc. But, in fact this tooltip contains a strict structure and is designed only to show some statistics.

Let's rename it please.

2. I'm also a little confused by the fact that we have a components/Tooltip/TooltipCard structure that contains a very specialized TooltipCard that is exported by default, and a TooltipCardRow that is also used in different places

If I were doing this, I would name the folder something more specific, like components/PoolStatsTooltip/PoolStatsTooltip.tsx (not sure, you can probably think of a better one)

3. It seems to me that it is better not to mix different kinds of imports unnecessarily. I personally would do everything as an export function, but it's up to you.)

Reply:
Makes sense! Refactored